pkg/api/types.go

```diff
@@ -650,6 +650,9 @@ type TestStepConfiguration struct {
 	// RunIfChanged is a regex that will result in the test only running if something that matches it was changed.
 	RunIfChanged string `json:"run_if_changed,omitempty"`
 
+	// Optional defines if the job will be marked as optional or required.
```

Comment:
Such Godoc really has no added value over the name of the field, please make it more precise (good sentences will probably mention something about merging)